cmd/dcrtime/dcrtime.go

```diff
@@ -936,6 +938,112 @@ func showWalletBalanceV2() error {
 	return nil
 }
 
+// lastAnchorV1 returns the last anchor information
+// such as: tx, chain timestamp, block height & block hash
+func lastAnchorV1() error {
+	c := newClient(*skipVerify)
+	route := *host + v1.LastAnchorRoute
+
+	fmt.Println(route)
```

Comment:
Aren't we only suppose to be printing routes when the verbose flag is used?  What do the other commands do?

Same comment for v2 command.

Reply:
Also, this is being printed even when the json flag is used.  Is this what the other commands do? Typically, you only want to print only json when the json flag is used so that you can pipe it into other commands like jq.

Reply:
Checking, I remember copying this from other route.

Reply:
@lukebp 
> Aren't we only suppose to be printing routes when the verbose flag is used? What do the other commands do? 

Just checked, all other routes print route only if -debug flag is provided.
Which also answer your second question - now `-json` would print the res as json without any other info.